Doc/library/io.rst

```diff
@@ -168,6 +168,19 @@ High-level Module Interface
    :func:`os.stat`) if possible.
 
 
+.. data:: SEEK_SET
+          SEEK_CUR
+          SEEK_END
+
+   Parameters to the :meth:`~IOBase.seek` method,
+   used to specify the relative position
+   Their values are 0, 1, and 2, respectively.
+
+   .. seealso::
+
+      :data:`os.SEEK_SET`, :data:`os.SEEK_CUR`, and :data:`os.SEEK_END`.
```

Comment:
It seems the ``os`` constants were added in 2.5 for the ``os.lseek`` function (and ``io``'s in 2.7 & 3.1 (#48822)). Given the values are the same there's little difference, but should we provide which module's constants you're 'meant' to use in which cases?

Edit: Note that ``fnctl.lockf``, ``mmap.seek`` and ``sqlite3.Blob.seek`` use the ``os`` constants.

Reply:
I have no opinion on which constants to use, but my gut reaction would be to use the constants from `io` for `io` stuff.